Applications/Spire/Include/Spire/Ui/RecentColors.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef SPIRE_RECENT_COLORS_HPP
+#define SPIRE_RECENT_COLORS_HPP
+#include <array>
+#include <deque>
+#include <QColor>
+
+namespace Spire {
+
+  //! Represents a data structure for storing recently selected colors in a
+  //  FIFO manner.
+  class RecentColors {
+    public:
+
+      //! The maximum number of recent colors to store.
+      static const int RECENT_COLOR_COUNT = 9;
```

Comment:
Use ```auto``` even for ```static``` fields.

Use ```constexpr``` if you want to access that field in a constant expression, such as in ```std::array```.

The end result is:

    static constexpr auto RECENT_COLOR_COUNT = 9;